sphinx_gallery/gen_rst.py

```diff
@@ -140,8 +140,6 @@ def __exit__(self, type_, value, tb):
 
 CODE_OUTPUT = """.. rst-class:: sphx-glr-script-out
 
- Out:
-
```

Comment:
The only disadvantage I can think of here is that this will render differently (without the `Out:`) in non-HTML builds like LaTeX for example. But I doubt that matters too much -- the gain for HTML builds is probably worth the small LaTeX loss.

Reply:
I think that's okay, as a) the 'Out:' text is in the normal font and doesn't help to call out the output, and b) the output is in a frame already.